src/styles/theming/sap_fiori_3.scss

```diff
@@ -263,4 +263,10 @@
 
   /* Micro Process Flow */
   --fdMicro_Process_Flow_Icon_Color: var(--sapContent_ContrastIconColor);
+
+  /* Scrollbar */
+  --fdScrollbarTrackColor: var(--sapScrollBar_TrackColor);
+  --fdScrollbarThumbColor: var(--sapScrollBar_FaceColor);
+  --fdScrollbarThumbHoverColor: var(--sapScrollBar_Hover_FaceColor);
+  --fdScrollbarDimension: var(--sapScrollBar_Dimension);
```

Comment:
That's not the proper naming we have adopted. Please use:
--fdScrollbar_Track_Color: var(--sapScrollBar_TrackColor);
--fdScrollbar_Thumb_Color: var(--sapScrollBar_FaceColor);
--fdScrollbar_Thumb_Hover_Color: var(--sapScrollBar_Hover_FaceColor);
--fdScrollbar_Dimension: var(--sapScrollBar_Dimension);